hasura/metadata/databases/ibcjuno/tables/public_token_unit.yaml

```
@@ -0,0 +1,43 @@
+table:
+  name: token_unit
+  schema: public
+object_relationships:
+- name: token
+  using:
+    foreign_key_constraint_on: token_name
+- name: token_price
+  using:
+    manual_configuration:
+      column_mapping:
+        denom: unit_name
+      insertion_order: null
+      remote_table:
+        name: token_price
+        schema: public
+- name: token_ibc_denom
+  using:
+    manual_configuration:
+      column_mapping:
+        denom: denom
+      insertion_order: null
+      remote_table:
+        name: token_ibc_denom
+        schema: public
+array_relationships:
+- name: token_ibc_denom
```

I think we can change this to be simply `ibc_denoms` for easier usage 
```suggestion
- name: ibc_denoms
```